### PR TITLE
Kwargs expansion, linting, and formatting

### PR DESCRIPTION
**Pull request by Robert-Zacchigna**
Hello again, so it seems I was finally motivated enough to expand and lint `kwargs` for the etrade api. 

**THIS WILL BE A BREAKING CHANGE**

I expanded all methods using `kwargs` (except in **order.py**, that will have to be done another day...) with the api variables and their respective defaults.

Since before you had to use the specific name of the api variable to be expanded by kwargs, the variable names had to be slightly changed to fit python formatting standards.

**EXAMPLES:** 
* `sortBy` becomes `sort_by`
* `startDate` becomes `start_date`
* `sortOrder` becomes `sort_order`
* etc...

So anyone using kwargs before this point will need to update their variable names to match the ones now added to the methods.

I also applied some spelling, grammar and other general formatting fixes.

@jessecooper @1rocketdude Please review and let me know if anything needs to be changed/fixed. This should hopefully make understanding and using the various methods much easier for people.

### CODE REVIEW

**Comment by jessecooper** [comment]
@Robert-Zacchigna Thanks for the PR and apologies for the lack of response. Give me some time to review and test these changes.

**Comment by Robert-Zacchigna** [comment]
@jessecooper just checking in, any issues and/or changes you think i need to make?

**Comment by jessecooper** [comment]
I apologize I have not had a moment to give this a full review. It is next on my backlog. I have not forgotten I promise.

**Comment by csinko** [comment]
> THIS WILL BE A BREAKING CHANGE

I would bump version to at least 1.5 then

File: setup.py
Comment (jessecooper) [inline setup.py:7]: Since this has breaking changes I am going to bump `pyetrade` to `2.0.0`. We have two options here either you can bump it using `bumpversion`  or just change this back to `1.4.1` and I will merge it and bump it after. It wont clobber what is currently on PyPI.

**Comment by jessecooper** [comment]
@Robert-Zacchigna This is a great PR. Thank you for taking your time to make these much needed changes and also cleaning things up and added type annotations. It is very appreciated. Please see comment: https://github.com/jessecooper/pyetrade/pull/89/files#r1526233328. Either you can just change the file back and I can handle the bump after the merge or you can do the bump either way will work for me.

File: setup.py
Comment (Robert-Zacchigna) [reply setup.py]: i reverted the version bump, see latest commit

**Comment by Robert-Zacchigna** [comment]
> @Robert-Zacchigna This is a great PR. Thank you for taking your time to make these much needed changes and also cleaning things up and added type annotations. It is very appreciated. Please see comment: https://github.com/jessecooper/pyetrade/pull/89/files#r1526233328. Either you can just change the file back and I can handle the bump after the merge or you can do the bump either way will work for me.

No problem at all, I'm glad you found the PR to be of good quality.

I have reverted my version bump, you are free to handle that as you see fit.

I'm sure I'll mosey on back to this again to finish `order.py` when i have more free time again (if you don't beat me to do it).

**Review by jessecooper — APPROVED** [review]
LGTM;

**Comment by jessecooper** [comment]
This is released in pyetrade `2.0.1`

**Pull request merged**
